test/runtests.jl

```diff
@@ -1,136 +1,143 @@
 using SyntheticGrids
-using Base.Test
+using Test
 using JSON
 
 @testset "SyntheticGrids" begin
-    const SEED = 666
-    const GENPATH = joinpath(dirname(@__FILE__), "..", "data", "GenData.json")
-    const CENSUSPATH = joinpath(dirname(@__FILE__), "..", "data", "Census_data.dat")
-    const GENCOORDPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_coord.dat")
-    const GENDATAPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_data.dat")
-    const DUMMYPATH = joinpath(dirname(@__FILE__), "..", "data", "dummy.json")
-    const DUMMYPPC = joinpath(dirname(@__FILE__), "..", "data", "dummy.p")
-    const GRIDPATH = joinpath(dirname(@__FILE__), "..", "data", "testgrid.json")
+    SEED = 666
+    LATLIM = (38, 40)
+    LONGLIM = (-89, -88)
+    GENPATH = joinpath(dirname(@__FILE__), "..", "data", "GenData.json")
```

Comment:
You can replace `dirname(@__FILE__)` with `@__DIR__`:
```suggestion
    GENPATH = joinpath(@__DIR__, "..", "data", "GenData.json")
```